frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/moderation/ModerationPortlet.java

```diff
@@ -57,11 +58,27 @@
     @Override
     public void serveResource(ResourceRequest request, ResourceResponse response) throws IOException, PortletException {
         String action = request.getParameter(PortalConstants.ACTION);
-        if (isGenericAction(action)) {
+        if (PortalConstants.ACTION_REMOVEME.equals(action)){
+            removeMeFromModerators(request, response);
+        } else if (isGenericAction(action)) {
             dealWithGenericAction(request, response, action);
         }
     }
 
+    private void removeMeFromModerators(ResourceRequest request, ResourceResponse response){
+        final User user = UserCacheHolder.getUserFromRequest(request);
+        final String id = request.getParameter(MODERATION_ID);
+        ModerationService.Iface client = thriftClients.makeModerationClient();
+        RemoveModeratorRequestStatus status = null;
+        try {
+            status = client.removeUserFromAssignees(id, user);
+        } catch(TException e) {
+            log.error("Error in Moderation ", e);
+        }
+        request.setAttribute(PortalConstants.REQUEST_STATUS, status);
```

Comment:
Would be good to move this into the `try`-block. One should not set an attribute to `null`